config.go

```diff
@@ -120,12 +119,8 @@ var (
 
 func init() {
 	AppPath, _ = filepath.Abs(filepath.Dir(os.Args[0]))
-	workPath, _ = os.Getwd()
```

Comment:
should change to AppPath for some configure files,and workPath is useless in fact.